docs/beta.md

```diff
@@ -3,6 +3,9 @@
   **WARNING**: Do not rely on BETA features in production workflows.
   Support from Tamr may be limited.
 
+## Tutorials
+  * [Get Tamr version](beta/tutorial/get_version)
```

Comment:
Can we have a link to the [beta documentation](https://tamr-client.readthedocs.io/en/stable/beta.html) somewhere in this tutorial?

Reply:
With the current Doc structure (which will change at some point) this tutorial is an immediate child of the beta documentation page, so the nav bar on the left serves the purpose of such a link.

Reply:
Got it! Thanks @skalish